plugin/trino-prometheus/src/main/java/io/trino/plugin/prometheus/PrometheusSplitManager.java

```diff
@@ -87,6 +83,10 @@ public ConnectorSplitSource getSplits(
         if (table == null) {
             throw new TableNotFoundException(tableHandle.toSchemaTableName());
         }
+
+        Duration maxQueryRangeDuration = PrometheusSessionProperties.getMaxQueryRange(session);
+        Duration queryChunkSizeDuration = PrometheusSessionProperties.getQueryChunkSize(session);
```

Comment:
nit: Static import getMaxQueryRange and getQueryChunkSize methods.

Reply:
Done